integration

```diff
@@ -334,6 +334,7 @@
   "fineemb/Xiaomi-Smart-Multipurpose-Kettle",
   "finity69x2/nws_alerts",
   "firstof9/ha-openei",
+  "Flix6x/flexmeasures-ha-integration",
```

Comment:
This is a fork:
- That is behind upstream
- Does not add anything that is not in upstream

For those 2 reasons Im not going to accept this.

Reply:
I understand, but I could use some instructions. [You closed](https://github.com/hacs/default/pull/2164#discussion_r1485630021) our previous PR for not being editable. Therefore, I had created this private fork from our organization, because [the docs suggest](https://hacs.xyz/docs/publish/include/#before-submitting) `Your PR needs to be editable, so you can not submit it from an organization.` What should I do, orphan the fork / start over?

Reply:
That is about _this_ repo, not what you are submitting.

Reply:
I don't understand, what should I be doing?

Reply:
The same as you did here, using your personal fork of this repository, but you add the repository you want (belong to the org and not your outdated fork)